src/flowTypes.js

```diff
@@ -249,3 +249,10 @@ export type Notifications = {
 };
 
 export type UserAchievementsMap = {};
+
+export type TraitClick = {
+  id: number,
+  isMajorTrait: boolean,
```

Comment:
what would a better api be, passing a boolean true or false for major trait, or a string that is either `minor` or `major`?

Reply:
Having come more from the Java world rather than JavaScript, I have a decent aversion to stringly typed variables, but I suppose it would have been better.

However, come to think of it, I'm not entirely sure why I added it there in the first place, I should be able to get that data from the store instead, so I think I'll actually just switch to pure number for the id as a parameter.